executor/show.go

```diff
@@ -429,7 +430,15 @@ func (e *ShowExec) fetchShowCreateTable() error {
 				case "CURRENT_TIMESTAMP":
 					buf.WriteString(" DEFAULT CURRENT_TIMESTAMP")
 				default:
-					buf.WriteString(fmt.Sprintf(" DEFAULT '%v'", col.DefaultValue))
+					defaultValStr := fmt.Sprintf("%v", col.DefaultValue)
+					if col.Tp == mysql.TypeBit {
+						bytes := make([]byte, 8)
+						copy(bytes[8-len(defaultValStr):8], []byte(defaultValStr))
+						intValue := binary.BigEndian.Uint64(bytes)
```

Comment:
This piece of code can be reorganized better in https://github.com/pingcap/tidb/pull/4415. However as a hot fix for master, let's keep as it is.